scripts/import-sdvx.js

```diff
@@ -91,7 +117,51 @@ async function main() {
   );
 }
 
-function buildSong(song) {
+function determineDiffClass(song, chartType) {
+  if (chartType !== "infinite") {
+    return chartType;
+  }
+  const infVersion = parseInt(song.info[0].inf_ver[0]._);
+  switch (infVersion) {
+    case 2:
+      return "infinite";
+    case 3:
+      return "gravity";
+    case 4:
+      return "heavenly";
+    case 5:
+      return "vivid";
+  }
+}
+
+function filterUnplayableSongs(song) {
+  const songsIdsToSkip = [
+    840,  // Grace's Tutorial https://remywiki.com/GRACE-chan_no_chou~zetsu!!_GRAVITY_kouza_w
+    1219, // Maxima's Tutorial https://remywiki.com/Maxima_sensei_no_mankai!!_HEAVENLY_kouza
+    1259, // AUTOMATION PARADISE
+    1438, // AUTOMATION PARADISE, April Fools
+  ];
+  return !songsIdsToSkip.some(songIdToSkip => parseInt(song.$.id) === songIdToSkip);
+}
+
+function determineChartJacket(chartType, song, availableJackets) {
+  const songId = ('000' + parseInt(song.$.id)).slice(-4);
+  const diffClassToNumber = {
+    "novice": 1,
+    "advanced": 2,
+    "exhaust": 3,
+    "infinite": 4,
+    "maximimum": 5
+  }
+  // if a chart does not have difficulty-specific song jackets, then they share the "novice" jacket
+  let jacketName = `jk_${songId}_${diffClassToNumber[chartType]}_s.png`;
+  if (!availableJackets.some(j => j === jacketName)) {
+    jacketName = `jk_${songId}_${diffClassToNumber["novice"]}_s.png`;
+  }
+  return `sdvx/${jacketName}`;
```

Comment:
Seems like we return a jacket path for every chart, regardless of whether it has a unique jacket or not. This seems like it will end up bloating the size of final data file. Instead, can we just return null/undefined in the case it doesn't have a unique jacket and set the novice chart at the song-level (Line 95)?